pandas/_libs/tslibs/c_timestamp.pyx

```diff
@@ -54,12 +54,12 @@ class NullFrequencyError(ValueError):
 def maybe_integer_op_deprecated(obj):
     # GH#22535 add/sub of integers and int-arrays is deprecated
     if obj.freq is not None:
-        warnings.warn("Addition/subtraction of integers and integer-arrays "
-                      "to {cls} is deprecated, will be removed in a future "
-                      "version.  Instead of adding/subtracting `n`, use "
-                      "`n * self.freq`"
-                      .format(cls=type(obj).__name__),
-                      FutureWarning)
+        warnings.warn(f"Addition/subtraction of integers and integer-arrays "
+                      f"to {type(obj).__name__} is deprecated, "
+                      f"will be removed in a future "
+                      f"version.  Instead of adding/subtracting `n`, use "
+                      f"`n * self.freq`"
```

Comment:
@datapythonista was the decision that we are or are not putting the "f" one every line?

Reply:
IIRC that was the preferred option. To me it makes more sense.

Reply:
@datapythonista Do you think that it's something that is worth mentioning in the docs?
I mean, to sort the confusion once and for all?